CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -211,7 +211,9 @@ public function buildQuickForm($check = FALSE) {
     // is this processor active ?
     $this->add('checkbox', 'is_active', ts('Is this Payment Processor active?'));
     $this->add('checkbox', 'is_default', ts('Is this Payment Processor the default?'));
-
+    $creditCardTypes = CRM_Contribute_PseudoConstant::creditCard();
+    $this->addCheckBox('accept_credit_cards', ts('Select Credit Card Types that this payment processor can accept'),
```

Comment:
seems to be a long label for a field. Can we not shorten this to something `Accepted Credit Card Type(s)` or similar and move the current label to description ?